zebra-network/proptest-regressions/peer_set/set/tests/prop.txt

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
+#
+# It is recommended to check this file in to source control so that
+# everyone who runs the test benefits from these saved cases.
+cc a1098eb7e8d9970d8624949e224c398cc3f522a3da8669906cda4dbe27977da8 # shrinks to total_number_of_peers = 2
```

Comment:
This proptest failure seems unrelated to this change, can we leave it out?

(Sometimes the network tests panic when `cargo watch` restarts a test, when the network connection is unreliable, or when there are multiple tests running at the same time.)